src/main/java/racingcar/Main.java

```diff
@@ -3,20 +3,18 @@
 import racingcar.view.InputView;
 import racingcar.view.ResultView;
 
-
 public class Main {
 
     public static void main(String[] args) {
 
-        int carCount = InputView.getCarCount();
+        String[] carNames = InputView.getCarNames();
         int gameRound = InputView.getGameRound();
```

Comment:
```suggestion
        int gameRoundCount = InputView.getGameRoundCount();
```
가 더 명확한 명명이라고 생각하는데, 어떻게 생각하시나요?

Reply:
잘 캐치해주셔서 감사합니다.  게임라운드를 객체로오인할수 있는 함수명이네요. 더명확한 것 같습니다.